x/ccv/provider/keeper/consumer_lifecycle.go

```diff
@@ -330,8 +330,14 @@ func (k Keeper) MakeConsumerGenesis(
 			"consumerId", consumerId,
 			"minPower", minPower,
 		)
-		k.OptInTopNValidators(ctx, consumerId, activeValidators, minPower)
+
+		// set the minimal power of validators in the top N in the store
 		k.SetMinimumPowerInTopN(ctx, consumerId, minPower)
+
+		err = k.OptInTopNValidators(ctx, consumerId, activeValidators, minPower)
+		if err != nil {
```

Comment:
Can you log this error? I see that in the code below loging was removed. It would be great to have at least some indication of what's going on.

Reply:
The error ends up being logged in `BeginBlockLaunchConsumers` where it will just skip trying to launch the chain.